### PR TITLE
chore: list as the first child in TOC

### PR DESCRIPTION
**Pull request by g1eny0ung**
Signed-off-by: Yue Yang <g1enyy0ung@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

Related to https://github.com/pingcap/docs/pull/6852.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [x] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/TOC.md [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
-+ [快速上手](get-started.md)
-+ 部署
+- [快速上手](get-started.md)
+- 部署
   - 部署 TiDB 集群
     - [部署到 AWS EKS](deploy-on-aws-eks.md)
     - [部署到 GCP GKE](deploy-on-gcp-gke.md)
     - [部署到阿里云 ACK](deploy-on-alibaba-cloud.md)
-    + 部署到自托管的 Kubernetes
+    * 部署到自托管的 Kubernetes
```
Comment (TomShawn) [inline zh/TOC.md:13]: Use `-` for consistency?
```suggestion
    - 部署到自托管的 Kubernetes
```

Comment (g1eny0ung) [reply zh/TOC.md:13]: Ok. Will update soon.

Comment (g1eny0ung) [reply zh/TOC.md]: Fix in 6a399ad.

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**